wsmaster/che-core-api-workspace/src/main/resources/schema/2.0.0/devfile.json

```diff
@@ -1,8 +1,3374 @@
 {
-  "description": "This is a temporary dummy structure for devfile 2.0.0.",
+  "description": "Devfile describes the structure of a cloud-native workspace and development environment.",
```

Comment:
This isn't necessarily related to this PR but is it possible to make this schema available via an api so that we can consume it on the dashboard side?

Reply:
yes, please create an issue for it

Reply:
issue created https://github.com/eclipse/che/issues/19023